src/app/beer_garden/api/stomp/__init__.py

```diff
@@ -0,0 +1,96 @@
+# -*- coding: utf-8 -*-
+"""Package containing the Stomp entry point"""
+import logging
+import types
+
+import beer_garden.config as config
+from tornado.ioloop import IOLoop
+from brewtils.models import Event, Events
+import beer_garden.events
+import beer_garden.router
+
+from beer_garden.api.stomp.processors import EventManager
+from beer_garden.api.stomp.server import Connection
+from beer_garden.events import publish
+from beer_garden.events.processors import QueueListener
+
+logger = None
+conn = None
+
+io_loop = None
```

Comment:
The `io_loop` code shouldn't be necessary, and it may actually be causing some performance losses. `stomp.py` is not an asyncio thing, so there's no need to start up an event loop.

This should make things simpler than the HTTP entry point - everywhere that does something like `io_loop.add_callback` should just be able to call the function directly. For example, in the signal handler

```python
io_loop.add_callback_from_signal(shutdown)
```

can be 

```python
shutdown()
```

Reply:
I removed the `io_loop` and created a thread to get events from `ep_conn`